src/utility/Constants.js

```diff
@@ -101,6 +101,7 @@ export const hackerApplicationTemplate = Object.freeze({
 // relevant dates for each status message displayed on the dashboard
 export const relevantDates = Object.freeze({
   sendAcceptancesBy: 'December 30th, 2020',
-  rsvpBy: 'Janurary 7th at 11:59 PM (Pacific Time)',
+  rsvpBy: 'Janurary 3rd at 11:59 PM (Pacific Time)',
+  offWaitlist: 'one week before the event',
   hackathonDate: 'January 9-10th',
 })
```

Comment:
is there a reason these aren't on firebase? livesite has some fields related to these

Reply:
Oh I totally didn't realize that there were similar fields on firebase
We were just a little short on time and I needed to make this change really quickly 😢 but let me just manually add some new fields and reference from firebase instead

Reply:
Ok updated! Pulling from firebase now